daemon/libnetwork/drivers/bridge/internal/iptabler/cleaner.go

```diff
@@ -31,29 +33,33 @@ type iptablesCleaner struct {
 //
 // If there are no old rules to clean up, return nil.
 func NewCleaner(ctx context.Context, config firewaller.Config) firewaller.FirewallCleaner {
-	clean := func(ipv iptables.IPVersion, enabled bool) bool {
+	clean := func(ipv iptables.IPVersion, enabled bool) (bool, bool) {
```

Comment:
`(bool, bool)` 🙈 

This could use a helper struct to describe each bool.

(or at least use named returns)